### ISSUE TITLE
[Sumtree]: Track total bid and ask liquidity in state for queries

### ISSUE THREAD

**Issue by AlpinYukseloglu** [osmosis-labs/orderbook#139]
## Background

Our `GetTotalPoolLiquidity` query is currently prohibitively inefficient (iterates over all initialized ticks on each call). This is both wastefully expensive and likely just untenable for e.g. SQS usage.

## Suggested Design

Track total bid and ask liquidity in state:
* Every placed limit order should increment it
* Every cancelled limit order should decrement
* Every tick fill should decrement

Basically it should just mirror the state updates of `total_amount_of_liquidity` in tick state.

Then the CW pool query should be updated to simply read and return this value, making it O(1).

## Acceptance Criteria

* Total liquidity tracking is implemented and wired up to CW pool query
